File: tests/interfile/javascript/test_lambda_deeply_nested_js_split/test4.js
function test4() {
    let x = source();
    let level1 = () => {
        let level2 = () => {
            let level3 = () => {
                // ruleid: test-lambda-deeply-nested-js
                sink(x);
            };
            level3();
        };
        level2();
    };
    level1();
}

File: tests/interfile/rust/test_lambda_deeply_nested_rust_split/test4.rs
fn test4() {
    let x = source();
    let level1 = || {
        let level2 = || {
            let level3 = || {
                // ruleid: test-lambda-deeply-nested-rust
                sink(&x);
            };
            level3();
        };
        level2();
    };
    level1();
}
